### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Describe your changes here.

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/primer-react/commit/7c064561796648c1111cd02d564cf25bd81472ca#commitcomment-67584831

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/primer-react/commit/cd7b10ef0b0f7f356e7758505656841428ba3d49

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### Screenshots

Please provide before/after screenshots for any visual changes

### Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Tested in Chrome
- [ ] Tested in Firefox
- [ ] Tested in Safari
- [ ] Tested in Edge

Take a look at the [What we look for in reviews](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#what-we-look-for-in-reviews) section of the contributing guidelines for more information on how we review PRs.

### CODE REVIEW

File: contributor-docs/adrs/adr-002-behavior-isolation.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Building behaviors in React Hooks gives us the ability to provide things like st
 
 #### Other
 
-- The custom element and web component API progress slower than React due to changes needing to go through the whatwq standards process.
+- The custom element and web component API progress slower than React due to changes needing to go through the whatwg standards process.
```
Comment (jsoref) [inline contributor-docs/adrs/adr-002-behavior-isolation.md:62]: [brand](https://whatwg.org/)?

File: .storybook/preview.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ const withThemeProvider = (Story, context) => {
 export const decorators = [withThemeProvider, withPerformance]
 
 addons.setConfig({
-  // Some stories may set up keyboard event handlers, which can can be interfered
+  // Some stories may set up keyboard event handlers, which can be interfered
```
Comment (jsoref) [inline .storybook/preview.js:119]: This was caught by duplicate word detection, which is a fairly new feature in my tooling. Sometimes it's valuable, and sometimes it's annoying. I'm still gathering input based on experience...

File: contributor-docs/adrs/adr-002-behavior-isolation.md [hunk 3]
```diff
@@ -59,12 +59,12 @@ Building behaviors in React Hooks gives us the ability to provide things like st
...
 
 #### Risks of not switching to custom elements for behaviors
 
 - We spend extra time building behaviors in React that have already been built in our [custom elements library](https://github.github.io/web-systems-documentation/#custom-elements).
-  - There are currently 19 behaviors/components listed on the custom elements documentation site. Several of these we have already implemented in React in either Priemr React, Doctocat, or other React applications at GitHub which can be upstreamed (details-dialog, details-menu, clipboard-copy, text-expander, autocomplete, task-list via drag and drop hooks, tab-container, text-expander).
+  - There are currently 19 behaviors/components listed on the custom elements documentation site. Several of these we have already implemented in React in either Primer React, Doctocat, or other React applications at GitHub which can be upstreamed (details-dialog, details-menu, clipboard-copy, text-expander, autocomplete, task-list via drag and drop hooks, tab-container, text-expander).
```
Comment (jsoref) [inline contributor-docs/adrs/adr-002-behavior-isolation.md:67]: Brand

File: docs/content/theming.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ import deepmerge from 'deepmerge'
 
 const customTheme = deepmerge(theme, {
   fonts: {
-    mono: 'Monolisa, monospace'
+    mono: 'MonoLisa, monospace'
```
Comment (jsoref) [inline docs/content/theming.md:36]: [Brand](https://www.monolisa.dev/)

**Review by jsoref — COMMENTED** [review]
Most corrections were automatically suggested by Google Sheets.

All fault is mine...

**Comment by siddharthkp** [comment]
This is really cool, thank you! Added to our backlog to review all of them :)

**Review by colebemis — APPROVED** [review]
Thank you!

**Pull request merged**